src/AnalysisConfigFiles/RecognizerConfigFiles/Towsey.AnthusNovaeseelandiae.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+# Australasian Pipit = Towsey.Anthus novaeseelandiae
+# Resample rate must be 2 X the desired Nyquist
+ResampleRate: 16000
+# SegmentDuration: units=seconds;    
+SegmentDuration: 60
+# SegmentOverlap: units=seconds;
+SegmentOverlap: 0
+
+# Each of these profiles will be analyzed
+# This profile is required for the species-specific recogniser and must have the current name.
+Profiles:
+    PipitSyllable: !UpwardTrackParameters
+        ComponentName: Whip 
+        SpeciesName: AnthusNovaeseelandiae
+        FrameSize: 512
+        FrameStep: 256
+        WindowFunction: HANNING
+        # min and max of the freq band to search
+        MinHertz: 2000          
+        MaxHertz: 7000
+        MinBandwidthHertz: 500
+        MaxBandwidthHertz: 5000
+        DecibelThreshold: 9.0
+
+#################### POST-PROCESSING of EVENTS ###################
+
+# A: First post-processing steps are to combine overlapping/proximal/sequential events 
+#  1: Combine overlapping events
+CombineOverlappingEvents: true
+
+#  2: Combine syllables that possibly belong to the same strophe.
+#     Can also use this to "mop up" events in neighbourhood - these can be removed later. 
+CombinePossibleSyllableSequence: true
+SyllableStartDifference: 0.25
+SyllableHertzGap: 3000
+
+#  B: Filter the events for excess activity in their upper and lower buffer zones
+NeighbourhoodLowerHertzBuffer: 200
+NeighbourhoodUpperHertzBuffer: 0
+NeighbourhoodDbThreshold: 9.0
+
+# C: Options to save results files
+#  4: Available options for saving spectrograms (case-sensitive): [False/Never | True/Always | WhenEventsDetected]
+# "True" is useful when debugging but "WhenEventsDetected" is required for operational use.
+#SaveSonogramImages: True
+SaveSonogramImages: WhenEventsDetected
```

Comment:
just as an FYI the format of these settings may change later

Reply:
True